elements/pfe-clipboard/demo/pfe-clipboard.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="/core/pfe-styles/red-hat-font.min.css">
+<link rel="stylesheet" href="/core/pfe-core/pfe.min.css">
+<link rel="stylesheet" href="/core/pfe-styles/pf-vars-rem.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-base.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-context.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-layouts.css">
+
+<style>
+  h2 {
+    margin-top: 20px;
+    margin-bottom: 0;
+  }
+</style>
+
+<pfe-band color="lightest">
+  <p>This element embeds a copy url button on your site.</p>
+
+  <h2>Clipboard</h2>
+  <pfe-clipboard role="button" tabindex="0"></pfe-clipboard>
+
+  <h2>Clipboard with no icon</h2>
+  <pfe-clipboard no-icon role="button" tabindex="0"></pfe-clipboard>
+
+  <h2>Clipboard: with custom icon</h2>
+  <pfe-clipboard role="button" tabindex="0">
+    <pfe-icon slot="icon" icon="web-icon-globe"></pfe-icon>
+  </pfe-clipboard>
+
+  <h2>Clipboard: with custom text & copying text from element</h2>
+  <pfe-clipboard role="button" tabindex="0" copy-from="#textToCopy">
+    <span slot="text">This will copy the text in the text field below!</span>
+    <span slot="text--success">Making some copies!</span>
```

Comment:
I think we can find and replace these instances of `text--success` with the new `success-text` slots in our demos.

Reply:
ack

should we rename `success-text` to just `success`?

Reply:
`success` is good.  Trying to un-BEM my brain. :P

Reply:
should we rename `text` to `label`? 

I might think `slot="text"` is the text-to-be-copied

Reply:
Makes sense 👍 

note: we'll have to update the slot names in the `docs` also, in addition to the `demo`.

Reply:
done